### DISCUSSION TITLE
tinygrad docs

### DISCUSSION THREAD

**Discussion by mustafaaljadery** [tinygrad/tinygrad#1719] [Show and tell]
I documented all of the tensor and nn functions in tinygrad. I provided examples for how each function is used. I also provided explanations of what each function does. I'm going to add paramters and types in the next couple of days. Moreover, I'm going to add a lot more functions. Let me know what you guys think

[https://tinygrad.vercel.app/](https://tinygrad.vercel.app/)

**Comment by wozeparrot** [comment]
I have docs going here as well: https://github.com/wozeparrot/tinygrad/blob/api_docs/docs/api/tinygrad/tensor.md

So, the main thing we are looking for in docs is that they need to be tested in CI or else they will go stale.
